### PR TITLE
User guide for capsule updates

### PR DESCRIPTION
**Pull request by SergiiDmytruk**

### CODE REVIEW

File: docs/guides/capsule-update.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+   Once the file-system is identified, run its name to switch to it:
+   ![Change file-system in UEFI Shell](../images/uefi-shell-cd-fs.png)
+
+6. If files are in file-system's root, no need to do anything here.  Otherwise,
+   use `cd` command to open a target directory to not have to enter full paths:
+   ![Change directory in UEFI Shell](../images/uefi-shell-cd-dir.png)
+
+7. Now you should be in a position to initiate a capsule update (run `ls` again
+   if in doubt about current location) via `CapsuleApp.efi
+   firmware.cap` (substitute `firmware.cap` with your an actual file name):
```
Comment (philipandag) [inline docs/guides/capsule-update.md:97]:
```suggestion
   firmware.cap` (substitute `firmware.cap` with an actual file name):
```

**Review by philipandag — COMMENTED** [review]
Looks good to me apart from this little error. Instructions work on qemu. Is it enough?

File: docs/guides/capsule-update.md (hunk 1 above)
Comment (SergiiDmytruk) [reply docs/guides/capsule-update.md]: https://github.com/Dasharo/docs/compare/cc990cc31ebdc1ba71d960ea4223de5b27b91a63..313559477b2072392c9ac465cde2263dc4e23a10

**Review by BeataZdunczyk — APPROVED** [review]

**Comment by BeataZdunczyk** [comment]
@SergiiDmytruk Nicely done, the render looks great. You can merge this PR after making pre-commit happy.

**Pull request merged**